extensions/smallrye-openapi/deployment/src/test/java/io/quarkus/smallrye/openapi/test/jaxrs/OpenApiResourceSecuredAtMethodLevel.java

```diff
@@ -37,6 +43,10 @@ public String secureEndpoint1() {
         return "secret";
     }
 
+    @APIResponses({
+            @APIResponse(responseCode = "401", description = "Who are you?"),
+            @APIResponse(responseCode = "403", description = "You cannot do that.")
+    })
```

Comment:
same comment as above

Reply:
Ok, sounds good;  I'll try to add those later today.

Reply:
@phillip-kruger I've updated the PR with separated out REST endpoints that are annotated to document the 401 and 403 responses, and a separate test method that just tests the 401 and 403 documentation.